src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -119,4 +131,17 @@ export default class FiatConnectQuote extends NormalizedQuote {
   getCryptoType(): CryptoType {
     return this.quote.quote.cryptoType
   }
+
+  getFiatAccountType(): FiatAccountType {
+    return this.fiatAccountType
+  }
+
+  getFiatAccountSchema(): FiatAccountSchema {
+    const fiatAccountSchemas = this.quote.fiatAccount[this.fiatAccountType]?.fiatAccountSchemas.map(
```

Comment:
```suggestion
    // NOTE: since we only support 1 fiat account schema right now, this is hardcoded to use a single fiat account. (Providers might support multiple fiat account schemas for the same quote.)
    const fiatAccountSchemas = this.quote.fiatAccount[this.fiatAccountType]?.fiatAccountSchemas.map(
```